ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -2992,6 +2996,8 @@
 				GCC_NO_COMMON_BLOCKS = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				PRODUCT_BUNDLE_IDENTIFIER = "org.reactivecocoa.$(PRODUCT_NAME:rfc1034identifier)-Tests";
+				PRODUCT_NAME = "$(PROJECT_NAME)";
```

Comment:
> The thing that is broken is that ENABLE_BITCODE is enabled for the OS X targets

This needs to be explicitly disabled for the mac tests. It already is for the framework